### DISCUSSION TITLE
Newbie here, a couple of questions to understand leantime better

### DISCUSSION THREAD

**Discussion by creactiv** [Leantime/leantime#724] [Q&A]
Well, 

after installing, getting over the obstacle of missing email in config file and doing first steps with leantime, i have a couple of questions: 

1. If i create a sub-task, and fill every data in, i cannot select it in timesheets. I can only select the main task in order to create the time tracking entry. If so, i'd suggest, being able to select a Task, and then a subtask to create an time tracking entry. I assume, that the time, i spent there will be calculated in the maintask as well. 
2. i've created a couple of tasks, and given them an estimated end date. I still lack milestones, but it would make sense, to see the gantt chart with the created tasks to see, whether they would fit on the timeline, or might be overlapping at some point and to create an overview for the customer, before we define milestones. 
3. Can we also export xlsx files, additional to .csv, please?
4. Whereas i can move the milestone in the Ganttdisplay forth and back, i cannot move it up or down; i'd like to have my own sorting (ie: training, project management and the like on the bottom. )
5. I installed source on one of my webhosts, and had the problem, that letsencrypt wasn't able to write into the public directory. I had to deactivate .htaccess for that moment, so the verification went through. Any expression i can add to .htaccess to allow writing in the challenge folder?
6. Timesheets: I would like to have a field showing not only how much hrs are left for a task but also for the project itself. Given that; i am wondering about the budget figure. I haven't found an option to change the dropdown lists about categorizing work, to add or delete entries. It would be nice, if i could add hourly rates, so i also see, how budget is depleting. If i do 4 days consulting, this would be as much as 5 days of programming, so i could do 20% more work with the same budget. 
7. Brilliant would be an Outlook Connector, so i can transfer bookings forth and back. 

As i wrote this, i remember seen something for timesheets, where i could delegate a subtask to another team member. That would be brilliant, because this is really needed. 

Let me close this one with a short statement for leantime: 

I have checked out various project management solutions, within CRM systems, based or not based on Redmine, they all lacked ease of use, design and with their freemium models one couldn't really test everything thoroughly before paying. So after a long search, i came here and i have to say that i am extremely pleased. 

For my projects this is near perfect. Why only near? I would like to stack projects, so i can have reports broken down. 
Customer A
-Implem,entation Project
-- Change Requests in 2021
-- Change Requests in 2022
- MVP
- -Change Requests in 2021
- -Change Requests in 2022
- Phase 2
- Support
- - 2021
- - 2022
So i would need the figures for Support in 2021 and 2022 and the total for Support as well as for change requests and the like.  If i could link those three 2021 entries together, this would also be brilliant. 

I know, that i am not the only one having ideas for leantime; unfortunately i am not a programmer, therefore i am not able to help on that side. 

Thanks for reading and best
c

**Comment by gloriafolaron** [comment]
@creactiv I’m about to start working on a more specific roadmap for leantime moving forward.  I feel like I’m reading more feature requests within your questions / comments along with your part at the end (also, thanks for the feedback!)

I’m not able to answer your questions at the moment but want to call this out quickly while I have a second — if you had some time, could you pull out your specific requests and throw them in a reply in a priority order that’d be most helpful to your workflows?
